libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -8,15 +8,34 @@
       <div class="reading-list-item--details--author">
         {{ b.authors.join(',') }}
       </div>
+      <div *ngIf="(b.finished)" class="finished" data-testing="reading-list-finished">
```

Comment:
Why the parentheses around `b.finished`?

Reply:
Not needed, removed the parenthesis.